inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/DataNodeEntity.java

```diff
@@ -32,6 +32,7 @@ public class DataNodeEntity implements Serializable {
     private Integer id;
     private String name;
     private String displayName;
+    private String tenant;
```

Comment:
Suggest moving the `tenant` field before the `inCharges` field.

Reply:
fixed, thx